openff/evaluator/protocols/forcefield.py

```diff
@@ -1100,6 +1100,10 @@ def _parameterize_molecule(self, molecule, force_field_source, cutoff):
 
         interchange = Interchange.from_foyer(topology=topology, force_field=force_field)
 
+        openmm_pdb_file = app.PDBFile(self.coordinate_file_path)
+        if openmm_pdb_file.topology.getPeriodicBoxVectors() is not None:
+            interchange.box = openmm_pdb_file.topology.getPeriodicBoxVectors()
+
```

Comment:
Does this mean `Interchange.from_foyer` isn't parsing the box vectors on the `topology` argument? If so, I think that's a bug

Reply:
`topology` is not an argument to `_parameterize_molecule`, just the `molecule` itself which contains no box information, the better fix would be to pass the topology to `_parameterize_molecule` but I'm trying to make the smallest changes I can

Reply:
the `topology` being passed to interchange is from `molecule.to_topology()` not from `TemplateBuildSystem._execute.topology` (which isn't passed as an argument to `_parameterize_molecule`)

Reply:
That makes sense, I was misreading the code then